src/instructlab/model/backends/backends.py

```diff
@@ -95,12 +95,19 @@ def is_model_gguf(model_path: pathlib.Path) -> bool:
         with model_path.open("rb") as f:
             first_four_bytes = f.read(4)
 
-        # Convert the first four bytes to an integer
-        first_four_bytes_int = int(struct.unpack("<I", first_four_bytes)[0])
+        try:
+            # Convert the first four bytes to an integer
+            first_four_bytes_int = int(struct.unpack("<I", first_four_bytes)[0])
+        except struct.error as e:
+            logger.error(f"Failed to unpack the first four bytes of {model_path}: {e}")
```

Comment:
This message might be a little obscure for the error level?  Maybe explain what we are looking for?